doc/ref/configuration/minion.rst

```diff
@@ -831,12 +831,28 @@ Default: ``False``
 
 The minion can locally cache grain data instead of refreshing the data
 each time the grain is referenced. By default this feature is disabled,
-to enable set grains_cache to ``True``.
+to enable set ``grains_cache`` to ``True``.
 
 .. code-block:: yaml
 
     grains_cache: False
 
+.. conf_minion:: grains_cache_expiration
+
+``grains_cache_expiration``
```

Comment:
this seems to not be related. Was this added on purpose?

Reply:
Nope.  Not sure how that got there.

Reply:
Updating to latest master resolved this.